libs/blocks/locui/loc/index.js

```diff
@@ -97,12 +97,20 @@ async function loadProjectSettings(projSettings) {
   }
 }
 
+function sanitize(url) {
+  const hlxUrl = /^https?:\/\/[^/?#]+/g;
+  const [hostname] = url.match(hlxUrl);
```

Comment:
Is there high certainty that the URL is never invalid? Otherwise this will return `null`, which will throw an error on the next line.

Reply:
There is a high certainty that this will always be a valid URL but I added a check to make sure. Thanks for spotting this!